xml/ha_hawk2_history_i.xml

```diff
@@ -385,7 +389,8 @@
     <listitem>
      <para>
       Shows a graphical representation of the transition. If you click
-      <guimenu>Graph</guimenu>, the PE is re-invoked (using the
+      <guimenu>Graph</guimenu>, <systemitem
+       class="daemon">pacemaker-schedulerd </systemitem> is re-invoked (using the
       <filename>pe-input*</filename> files), and generates a graphical
```

Comment:
It doesn't really call "pacemaker-schedulerd" here. It actually simulates the calculation as exactly done by pacemaker-schedulerd.

Reply:
Updated, thanks.